MPC/config/dds_no_suppress_anys.mpb

```diff
@@ -0,0 +1,4 @@
+project: dcps, dcps_ts_defaults {
```

Comment:
```suggestion
project: dcps_ts_defaults {
```
Does it work without the `dcps` base project?  Seems like it wouldn't be needed, also probably doesn't hurt to have it.